--- gngeo-0.8/src/blitter/opengl.c.orig	2015-02-24 12:31:08.251247174 +0000
+++ gngeo-0.8/src/blitter/opengl.c	2015-02-24 14:22:18.657477279 +0000
@@ -56,11 +56,14 @@
 	Uint32 width = visible_area.w;
 	Uint32 height = visible_area.h;		
 	
+        printf("OpenGL debug: blitter_opengl_init 0\n");
         if (load_glproc() == SDL_FALSE) return SDL_FALSE;
+        printf("OpenGL debug: blitter_opengl_init 1\n");
 
 	sdl_flags = (fullscreen?SDL_FULLSCREEN:0)| SDL_DOUBLEBUF | SDL_HWSURFACE
 	    | SDL_HWPALETTE | SDL_OPENGL | SDL_RESIZABLE;
 	SDL_GL_SetAttribute(SDL_GL_DOUBLEBUFFER, 1);
+  printf("OpenGL debug: blitter_opengl_init 2\n");
 
 	if ((effect[neffect].x_ratio!=2 || effect[neffect].y_ratio!=2) &&  
 	    (effect[neffect].x_ratio!=1 || effect[neffect].y_ratio!=1) ) {
@@ -75,6 +78,7 @@
 	    width *=effect[neffect].x_ratio;
 	    height*=effect[neffect].y_ratio;
 	}
+  printf("OpenGL debug: blitter_opengl_init 3\n");
 	width *= scale;
 	height *= scale;
 	printf("%d %d %d %d %d\n",width,height,scale,visible_area.w,visible_area.h);
@@ -88,7 +92,9 @@
 	conf.res_x=width;
 	conf.res_y=height;
 	
+  printf("OpenGL debug: blitter_opengl_init 4\n");
 	video_opengl = SDL_SetVideoMode(width, height, 16, sdl_flags);
+  printf("OpenGL debug: blitter_opengl_init 5\n");
 	
 	if ( video_opengl == NULL)
 		return SDL_FALSE;
@@ -101,7 +107,7 @@
 	
 	/* Linear Filter */
 	
-        pglTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+  pglTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
 	pglTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
 	/*
 	  pglTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
@@ -134,7 +140,7 @@
 	    c = (((float)visible_area.h/256.0))*effect[neffect].y_ratio/2.0;
 	    d = (((float)((visible_area.w<<1)-512)/256.0))*effect[neffect].y_ratio/2.0;
 	    screen = SDL_CreateRGBSurface(SDL_SWSURFACE, visible_area.w<<1,  /*visible_area.h<<1*/512, 16, 0xF800, 0x7E0, 0x1F, 0);
-	    //printf("[opengl] create_screen %p\n",screen);
+	    printf("[opengl] create_screen %p\n",screen);
 #ifdef USE_GL2
 	    tex_opengl= SDL_CreateRGBSurface(SDL_SWSURFACE, 1024, 512, 16, 0xF800, 0x7E0, 0x1F, 0);
 	    if (visible_area.w==320) {
@@ -163,8 +169,10 @@
 
   sdl_flags = SDL_DOUBLEBUF | SDL_HWSURFACE | SDL_HWPALETTE | SDL_OPENGL | SDL_RESIZABLE;
 
+  printf("OpenGL debug: blitter_opengl_resize 0\n");
   video_opengl = SDL_SetVideoMode(w, h, 16, sdl_flags);
 
+  printf("OpenGL debug: blitter_opengl_resize 1\n");
   if ( video_opengl == NULL)
     return SDL_FALSE;
 
@@ -177,13 +185,12 @@
 void 
 blitter_opengl_update()
 {
-#ifndef USE_GL2
+    printf("OpenGL debug: blitter_opengl_update 0 \n");
     float VALA=512/(float)visible_area.w-1.0; /* 240.0f*2.0f/304.0f - 1.0f */
-#endif
-//    int VALB=.0625; /* 16.0f/256.0f  */
-//    int VALC=.9375; /* 240.0f/256.0f */
+    int VALB=.0625; /* 16.0f/256.0f  */
+    int VALC=.9375; /* 240.0f/256.0f */
 //int VALD .25   /* 64.0f/256.0f */
-//    int VALD=1.0;
+    int VALD=1.0;
 
     if (neffect == 0) {
 #ifndef USE_GL2
@@ -353,20 +360,26 @@
 #endif
     }
 	
+    printf("OpenGL debug: blitter_opengl_update 1 \n");
     SDL_GL_SwapBuffers();	
+    printf("OpenGL debug: blitter_opengl_update 2 \n");
 }
 
 void
 blitter_opengl_close()
 {
-	//if (screen != NULL)
-	//	SDL_FreeSurface(screen);
+  printf("OpenGL debug: blitter_opengl_close 0 \n");
+	if (screen != NULL)
+    printf("OpenGL debug: blitter_opengl_close 1 \n");
+		//SDL_FreeSurface(screen);
 }
 
 void
 blitter_opengl_fullscreen()
 {
+  printf("OpenGL debug: blitter_opengl_fullscreen 0 \n");
 	SDL_WM_ToggleFullScreen(video_opengl);
+  printf("OpenGL debug: blitter_opengl_fullscreen 1 \n");
 }
 
 #endif
