--- src/Util.h.orig	2015-02-22 12:47:22.504134235 +0000
+++ src/Util.h	2015-02-22 12:47:25.740134347 +0000
@@ -23,6 +23,7 @@
 bool utilIsGzipFile(const char *);
 void utilStripDoubleExtension(const char *, char *);
 IMAGE_TYPE utilFindType(const char *);
+IMAGE_TYPE utilFindType(const char *, char (&)[2048]);
 u8 *utilLoad(const char *, bool (*)(const char*), u8 *, int &);
 
 void utilPutDword(u8 *, u32);
